internal/config/config.go

```diff
@@ -97,6 +97,7 @@ type Config struct {
 	ReplMonErrorWaitInterval                time.Duration                `config:"repl_mon_error_wait_interval" yaml:"repl_mon_error_wait_interval"`
 	ReplMonSlaveWaitInterval                time.Duration                `config:"repl_mon_slave_wait_interval" yaml:"repl_mon_slave_wait_interval"`
 	ShowOnlyGTIDDiff                        bool                         `config:"show_only_gtid_diff" yaml:"show_only_gtid_diff"`
+	ForceSwitchover                         bool                         `config:"force_switchover" yaml:"force_switchover"`
```

Comment:
Please, add TODO "Remove when we will be sure it's right way to do switchover" :)